src/shared/format-table.c

```diff
@@ -1089,10 +1089,11 @@ void table_set_cell_height_max(Table *t, size_t height) {
         t->cell_height_max = height;
 }
 
-int table_set_empty_string(Table *t, const char *empty) {
+void table_set_empty_string(Table *t, const char *empty) {
```

Comment:
hmm, i am a bit concerned about the fact that we almost always copy strings in cases like this, and here we do not.

Two ideas:

1. declare in the function name that this is a static string: `table_set_empty_string_static()` or so.

2. Maybe just add an enum instead, for the common types: 

```c
typedef enum EmptyCell {
        EMPTY_CELL_EMPTY,     /* → "" */
        EMPTY_CELL_NA,        /* → "n/a" */
        EMPTY_CELL_DASH,      /* → "-" */
        EMPTY_CELL_UNSET,     /* → "(unset)" */
        _EMPTY_CELL_MAX,
        _EMPTY_CELL_INVALID = -EINVAL;
} EmptyCell;
```

or so. Benefit: can be static string internally, but this does not leak into the API. Also: forces people to maybe stick to one of the established vocabularies...

Reply:
I kinda prefer the enum approach I must say

Reply:
this might even be useful later, because it takes up less space, in case we ever want to allow configuring this per column or even per-cell

Reply:
Updated.
I used different names though, because this is now used not only for replacement cells.